create.py

```diff
@@ -36,9 +36,11 @@
 with open('../portingdb/_check_drops/results.json', 'r') as f:
     results = json.load(f)
 
-with open('./bugz.json', 'r') as f:
-    bugz = json.load(f)
-
+# Get a list of components for which the bugs already exists
+tracking_bug = bzapi.getbug(int(TRACKER))
```

Comment:
I'm afraid I've broken this by having 2 numbers in tracker. 

```suggestion
tracking_bug = bzapi.getbug(int(TRACKER.split(',')[0]))
```